### DISCUSSION TITLE
[FeatureRequest] Add grafana_alerting_paused_alerts metric

### DISCUSSION THREAD

**Discussion by hartman17** [grafana/grafana#40999] [Alerting]
**What would you like to be added**:
By default Grafana is exposing a grafana_alerting_active_alerts metric, it would be beneficial to also report on paused alerts (new: grafana_alerting_paused_alerts (gauge)).  
**Why is this needed**:
This would allow alerting and visibility on the number of paused alerts in an instance.  This would help users who forget to "unpause" their alerts after a maintenance activity

**Comment by hugohaggmark** [comment]
Assigning this to @grafana/backend-platform for now.

**Comment by wbrowne** [comment]
@grafana/alerting-squad - looping you in here just so you're aware this exists

**Comment by gotjosh** [comment]
Pausing alerts is no longer a concept we have within unified alerting and given that we're deprecating legacy alerting soon I'd rather not include any extra features. 

In the new UI is very simple to know whether an alert is silenced or not being delivered - I'd encourage you to give it try and then report back with what could we improve within the scope of unified alerting.
